src/kOS/Module/kOSProcessor.cs

```diff
@@ -240,11 +240,14 @@ public ModifierChangeWhen GetModuleCostChangeWhen()
             return ModifierChangeWhen.FIXED;
         }
 
+        public float additionalMass = 0F;
```

Comment:
Defining this field should remain at the top of the file where it was previously instead of moving it.